### ISSUE TITLE
I would like to ask a question: What does the gated training data stream look like about this paper?

### ISSUE THREAD

**Issue by BOY-silverlinings** [yuvalatzmon/COSMO#2]
I would like to ask a question: What does the gated training data stream look like about this paper?
I would like to ask a question: What does the gated training data stream look like about this paper? Only input the trained ZSL and SEEN classifiers with known data, and use the obtained vectors to concatenate vectors to train gated? What is the final shape of the gated output? How should the loss label be designed?

**Comment by BOY-silverlinings** [comment]
Hope that there is a understanding of the great God to respond to comments, worship
